README.md

```diff
@@ -77,19 +77,15 @@ More about setting up environment with Docker Compose can be found [here](/docs/
 Below screenshots show how MedTagger looks like:
 
 #### Login Page
-![Login Page](/docs/assets/login_page.png)
+![Login Page](/docs/assets/redesigned/login-page.png)
```

Comment:
I would delete the old ones and replace them with your screenshots :)

Git will store the history for us! 💪